pvlib/tests/test_modelchain.py

```diff
@@ -170,6 +170,21 @@ def test_ModelChain_creation(sapm_dc_snl_ac_system, location):
     ModelChain(sapm_dc_snl_ac_system, location)
 
 
+def test_with_pvsyst(pvsyst_dc_snl_ac_system, location):
+    mc = ModelChain.with_pvsyst(pvsyst_dc_snl_ac_system, location)
+    assert mc.dc_model == mc.pvsyst
+
+
+def test_with_sapm(sapm_dc_snl_ac_system, location):
+    mc = ModelChain.with_sapm(sapm_dc_snl_ac_system, location)
+    assert mc.dc_model == mc.sapm
+
+
+def test_with_pvwatts(pvwatts_dc_pvwatts_ac_system, location):
+    mc = ModelChain.with_pvwatts(pvwatts_dc_pvwatts_ac_system, location)
+    assert mc.dc_model == mc.pvwatts_dc
```

Comment:
I think we may want to add `mc.run_model` to ensure the ModelChain object can execute it's method.